akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -490,7 +490,7 @@ private[akka] class ShardRegion(
     cluster.subscribe(self, classOf[MemberEvent])
     timers.startTimerWithFixedDelay(Retry, Retry, retryInterval)
     startRegistration()
-    if (settings.passivateIdleEntityAfter > Duration.Zero)
+    if (settings.passivateIdleEntityAfter > Duration.Zero && !settings.rememberEntities)
```

Comment:
These repeats of logic always seem brittle to me between `Shard` and `ShardRegion`. Perhaps a common function?